Shaders/deferred_light.frag.glsl

```diff
@@ -0,0 +1,533 @@
+#version 450
```

Comment:
Is this file used for anything? deferred_light.frag.glsl normally is at `Shaders/deferred_light/`